pebblo/app/config/config.py

```diff
@@ -47,7 +47,7 @@ def load_config(path, version) -> dict:
             version=version,
             daemon=PortConfig(host="localhost", port=8000),
             reports=ReportConfig(
-                format="pdf", renderer="xhtml2pdf", outputDir="~/.pebblo"
+                format="pdf", renderer="xhtml2pdf", cacheDir="~/.pebblo"
```

Comment:
is it default one ? what if there is no home dir ? are we creating this? what if we provide this through config.yaml ?

Reply:
- It is default on, if config file is not present.
- Whatever value we are passing in config.yaml it will be assigned to `cacheDir`
- If `cacheDir` (or `outputDir` for now) is not passed in config.yaml, current absolute path with be assigned to `cacheDir`